src/test/java/org/gridsuite/study/server/StudyTest.java

```diff
@@ -432,9 +433,9 @@ public void test() throws Exception {
                 .expectStatus().isOk()
                 .expectHeader().contentType(MediaType.APPLICATION_JSON)
                 .expectBodyList(StudyInfos.class)
-                .value(studies -> new MatcherStudyInfos(StudyInfos.builder().studyName("studyName").userId("userId").caseFormat("UCTE")
-                        .description("description").studyPrivate(false).creationDate(ZonedDateTime.now(ZoneId.of("UTC"))).loadFlowStatus(LoadFlowStatus.NOT_DONE)
-                        .build()).matchesSafely(studies.get(0)));
+                .value(studies -> assertTrue("response should match", new MatcherCreatedStudyBasicInfos<CreatedStudyBasicInfos>(CreatedStudyBasicInfos.builder().studyName("studyName").userId("userId").caseFormat("UCTE")
```

Comment:
Why `assertTrue` ?
We just need to pass a matcher to the value method of `WebTestClient`